src/main/kotlin/com/ajaxproject/telegrambot/bot/models/MongoUser.kt

```diff
@@ -0,0 +1,19 @@
+package com.ajaxproject.telegrambot.bot.models
+
+import com.ajaxproject.telegrambot.bot.models.MongoUser.Companion.COLLECTION_NAME
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.TypeAlias
+import org.springframework.data.mongodb.core.mapping.Document
+
+@TypeAlias("User")
+@Document(value = COLLECTION_NAME)
+data class MongoUser(
+    @Id
+    val id: Long,
```

Comment:
why id is long type here?

Reply:
Because I'm setting up an id through chatId which i receive from update:

```kotlin
 if (userService.getUserByPhoneNumber(phoneNumber) == null) {
            userService.addUser(MongoUser(chatId, phoneNumber))
        }
``` 
URL: https://github.com/SlavikN14/telegram-bot-bank/blob/945d413aa5773c55d409966cd05af2e0b37793ea/src/main/kotlin/com/ajaxproject/telegrambot/bot/handlers/impl/PhoneNumberHandler.kt#L45C36-L45C36